### ISSUE TITLE
Provide oci-systemd-hooks on minishift...

### ISSUE THREAD

**Issue by fbladilo** [minishift/minishift#1299]
Could we include the OCI systemd hooks as part of the minishift origin environment? The hooks assist the deployment of systemd based containers without the need for privileged SCC access.

Detailed project info : 

https://github.com/projectatomic/oci-systemd-hook

**Comment by gbraad** [comment]
since `oci-systemd-hook` is a package that is available from the repository, is this all that is needed?

**Comment by fbladilo** [comment]
@gbraad Tipically oci-systemd-hooks packages are installed on the OpenShift nodes/master which are usually VMs, I'm not sure how this applies to minishift. Do we need to install them on the busybox VM?

**Comment by anjannath** [comment]
Is this still relevent, i guess we have to bake our iso with the package??

**Comment by fbladilo** [comment]
@anjannath Yes it continues to be relevant for users deploying systemd based containers on minishift.

**Comment by gbraad** [comment]
Can you provide an example of a systemd based container that you would like
to deploy, and how this is used/deployed? We haven't seen other requests
for this... so we need to see why this matters.

**Comment by praveenkumar** [comment]
@fbladilo Can you provide details around @gbraad query?

**Comment by budhrg** [comment]
@fbladilo Can you provide more details as asked by @gbraad  here https://github.com/minishift/minishift/issues/1299#issuecomment-349154279?
